webpack.config.js

```diff
@@ -3,7 +3,6 @@
 var path = require('path');
 var webpack = require('webpack');
 var BundleTracker = require('webpack-bundle-tracker');
-var ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
This isn't compatible with Webpack 4. We were only using this for CSS, and there isn't any CSS left in the Webpack bundle.